modules/tile-converter/test/3d-tiles-converter/helpers/b3dm-converter.spec.js

```diff
@@ -100,7 +102,9 @@ test('tile-converter - b3dm converter#should convert material', async t => {
     const encodedContent = await b3dmConverter.convert(tile);
 
     const decodedContent = await load(encodedContent, Tiles3DLoader, {
-      isTileset: false,
+      '3d-tiles': {
```

Comment:
Nit: I think it could be nice if this option was false by default, so it only would need to be specified for a tileset.

Ideally we should be able to autodetect, but that is a separate problem